### DISCUSSION TITLE
Cant get jquery to work in Django+Tailwind

### DISCUSSION THREAD

**Discussion by metand-dotcom** [tailwindlabs/tailwindcss#2601] [Help] [answered]
Dear reader,

I'm new in webdev with the tailwind framework. And I like to know how to make Jquery work with tailwindcss examples
I installed django-tailwind and dont use the CDN.

My question is what links i'm missing in my template to access jquery part in the tailwindcss/ui part.
Can someone point me in the right directon, I dont like to use jquery but for most of the navbars its just needed.

it seems it also doesn't work on the one page playground:
https://play.tailwindcss.com/sNp6LSKKDC?file=config

//tailwind.config.js

  plugins: [
    require('@tailwindcss/custom-forms'),
    require('@tailwindcss/ui')({
      layout: 'sidebar',

    })
  ]

The only thing I load through the base.html is:

<link rel="stylesheet" href="{% static 'css/styles.css' %}">

//styles.
@tailwind base;
@tailwind components;
@tailwind utilities;

**Comment by simonswiss** [comment, marked as answer]
Hey @metand-dotcom 👋

Did you write some jQuery to toggle the navigation (on hamburger menu click) based on the following code comment?

```html
<!--
  Profile dropdown panel, show/hide based on dropdown state.

  Entering: "transition ease-out duration-100"
    From: "transform opacity-0 scale-95"
    To: "transform opacity-100 scale-100"
  Leaving: "transition ease-in duration-75"
    From: "transform opacity-100 scale-100"
    To: "transform opacity-0 scale-95"
-->
```

You might need to add a custom class/id/data-attribute to the elements you want to target with jQuery. It's tricky to help without seeing a codebase, do you have an example repo somewhere?

If you don't like using jQuery, I can recommend looking at [AlpineJS](https://github.com/alpinejs/alpine) for small UI interaction like toggling a navigation menu, it provides ma much simpler (and lightweight) implementation  👍

As a bonus, you can inspect the Tailwind UI demo examples with dev tools to see how we make it work 😅

**Reply by metand-dotcom** [reply]
It's clear to me the jQuery needs to be written by myself. That's not what I expect from sample components,because it still cost me time. I was hoping its embedded in the css that would access the jQuery part. Thanks for pointing me in the right direction and for the AlpineJS tip.
